src/app/Providers/components/ProvidersTable/VMware/VMwareProvidersTable.tsx

```diff
@@ -141,15 +143,14 @@ const VMwareProvidersTable: React.FunctionComponent<IVMwareProvidersTableProps>
     });
   });
 
+  const [isDownload, setDownload] = React.useReducer((isDownload) => !isDownload, false);
+
   return (
     <>
       <Level>
+        {isDownload ? <InventoryDownload providers={selectedItems} /> : null}
```

Comment:
Yeah, and then we wouldn't need `<InventoryDownload>` at all, the download function could just be in the `onClick` of the button